### PR TITLE
added catch block

### LINKED ISSUE #2: Handle exceptions instead of declaring them to be thrown
The line https://github.com/acmpesuecc/pinger/blob/master/src/main/java/pinger/Pinger.java#L11 declares that an `IOException` is thrown by the method. 

Instead of declaring it to be thrown, the entire method's contents should be wrapped in a `try {...} catch(...) {...}` statement to allow custom handling of the exception. The exception's stack trace should be printed along with some meaningful information about the exception.

### PR DESCRIPTION
**Pull request by sreechand2002**
resolved the issue

### CODE REVIEW

File: src/main/java/pinger/Pinger.java [hunk 1]
```diff
@@ -26,6 +26,8 @@ public static void main(String[] args) throws IOException {
                 addField("Header Fields", conn.getHeaderFields());
                 addField("Content", content);
             }
+        } catch(Exception e) {
+            System.out.println(e);
```
Comment (BoogieMonster1O1) [inline src/main/java/pinger/Pinger.java]: You can call `e.printStackTrace()` instead to print out the stack trace. Also print out a message saying that an error occured.

**Comment by sreechand2002** [comment]
fine i will make that changes

**Review by sreechand2002 — COMMENTED** [review]
changes made

**Comment by sreechand2002** [comment]
u can merge now i have made the changes

File: src/main/java/pinger/Pinger.java [hunk 2]
```diff
@@ -26,6 +26,9 @@ public static void main(String[] args) throws IOException {
                 addField("Header Fields", conn.getHeaderFields());
                 addField("Content", content);
             }
+        } catch(Exception e) {
+            System.out.println("error occured);
```
Comment (BoogieMonster1O1) [inline src/main/java/pinger/Pinger.java:30]: This doesn't compile
You can test your changes by running `./gradlew run`, or `gradlew.bat run`(windows)

**Comment by Rishabkashyap14** [comment]
!bounty 50

**Pull request closed without merging**